internal/exec/exec.go

```diff
@@ -37,7 +37,7 @@ type extensionser interface {
 }
 
 func makePanicError(value interface{}) *errors.QueryError {
-	return errors.Errorf("graphql: panic occurred: %v", value)
```

Comment:
This was causing panic errors to print like `graphql: graphql: ...` so I removed it. Not other tests failed, so I assume it was never checked.

Can revert is asked for it no problem.